chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/TriggerDeveloperBanJob.java

```diff
@@ -91,15 +91,9 @@ private String createHtmlEmailBody(JobExecutionContext jobContext) {
         }
         int openNcs = jdm.getInt("openNcs");
         int closedNcs = jdm.getInt("closedNcs");
-        String htmlMessage = String.format("<p>The CHPL Listing <a href=\"%s/#/product/%d\">%s</a>, owned by \"%s\" "
-                + "and certified by \"%s\" has been set on \"%s\" by \"%s\" to a Certification Status of \"%s\" with "
-                + "an effective date of \"%s\".</p>"
-                + "<p>%s</p>"
-                + "<p>%s</p>"
-                + "<p>There %s %d Open Nonconformit%s and %d Closed Nonconformit%s.</p>"
-                + "<p>ONC should review the activity and all details of the listing to determine if "
-                + "this action warrants a ban on the Developer.</p>",
+        String htmlMessage = String.format(env.getProperty("developerBanEmailBody"),
                 env.getProperty("chplUrlBegin"), // root of URL
+                env.getProperty("developerBanEmailUrl"),
```

Comment:
Why isn't this property just in the `developerBanEmailBody` string?

Reply:
yes. But Katy had comment to take URL as much as I can so I have it as separate. I can do either way. let me know which way to proceed

Reply:
Here's what I think: If the URL is something that's only in one place then it could be just put right in the HtmlMessage property in the properties file. This URL, however, is used in several places (including the three additional places in my first comment) so it could be kept as a separate property in the properties file but given a more generic name - maybe "listingDetailsUrl" and then used in this place plus the other three places that were changed.